sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/TopicRuntimeInfoExtensions.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections.Generic;
+
```

Comment:
nit: Move inside the namespace to keep it consistent

Reply:
Same in other classes

Reply:
Fixed